lib/game.rb

```diff
@@ -0,0 +1,43 @@
+class Game
+  def initialize
+    @rolls = [] 
+  end
+
+  def roll(pins)
+    @rolls.push(pins)
+  end
+
+  def points
```

Comment:
As far as I can tell, this seems to work perfectly. You can use the roll method to add too many rolls to the scorecard, but even so this method only counts rolls within a functional game and it seems to calculate correctly. I've tried to break it and I can't, great job